.babelrc

```diff
@@ -1,5 +1,13 @@
 {
   "presets": [
-    "shopify"
+    "env",
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
```

Comment:
What are these plugins for exactly?

I can appreciate `class-properties`.

Decorators I'm less convinced about. I know of them from looking at Typescript stuff, but I don't know if I just don't properly know what they do, or if I just don't quite get them yet?

What are `react-constant-elements` and `react-inline-elements` useful for?

Reply:
Honestly, I'm not sure – these were auto-populated when I installed (I presume) the extra babel dependencies needed. The react stuff can almost certainly be removed, as well as the decorators. Probably wouldn't harm putting back the `shopify` preset, either.

Reply:
Fair enough. I see no harm in keeping them in if I'm honest. It doesn't look like they break any existing functionality. I was hoping you could explain the benefit of decorators to me more than anything else. And personally I'd stick with the current presets. IIRC, they tend to be more up to date than the Shopify preset.